### ISSUE TITLE
[Accessibility]A11y_VSCode_SelectEncoding_Screenreader: Voiceover does not read the menulist item present in the tree view.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52311]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : MAC OS HighSierra10.13.5
**Additional Details:**
Voiceover,MAS1.3.1
**Repro Steps:**
1)Launch VSCode.
2)Navigate   to Status Bar and select "Select Encoding" button.
3)Check with Voiceover is reading.
**Actual:**
NVDA does not read the placeholder present on the screen.
**Expected:**
NVDA should read the placeholder which is present on the screen.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not able to know that there is placeholder present on the screen.
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
**Attachment for Reference:**
[SelectEncoding.pptx](https://github.com/Microsoft/vscode/files/2114801/SelectEncoding.pptx)

Does this issue occur when all extensions are disabled?: Yes
